File: Day15/ExcelColTitle.java
import java.util.Scanner;

public class ExcelColTitle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int columnNumber = sc.nextInt();
        ExcelColTitle s = new ExcelColTitle();
        System.out.println(s.convertToTitle(columnNumber));
    }

    public String convertToTitle(int columnNumber) {
        char[] lst = {'A','B','C','D','E','F','G','H','I','J','K','L','M',
                      'N','O','P','Q','R','S','T','U','V','W','X','Y','Z'};
        StringBuilder colTitle = new StringBuilder();
        int a = 26;
        while (columnNumber > 0) {
            columnNumber--;
            int b = columnNumber % a;
            colTitle.insert(0, lst[b]);
            columnNumber /= a;
        }
        return colTitle.toString();
    }
}

File: Day15/ExcelColTItle.py
class Solution(object):
    def convertToTitle(self, columnNumber):
        lst = ['A','B','C','D','E','F','G','H','I','J','K','L','M',
               'N','O','P','Q','R','S','T','U','V','W','X','Y','Z']
        col_title = ''
        a = 26
        while columnNumber > 0:
            columnNumber -= 1
            b = columnNumber % a
            col_title = lst[b] + col_title
            columnNumber //= a
        return col_title

if __name__ == "__main__":
    columnNumber = int(input())
    print(Solution().convertToTitle(columnNumber))
